.github/CONTRIBUTOR_AGREEMENT.md

```diff
@@ -87,7 +87,7 @@ U.S. Federal law. Any choice of law rules will not apply.
 7. Please place an “x” on one of the applicable statement below. Please do NOT
 mark both statements:
 
-    * [ ] I am signing on behalf of myself as an individual and no other person
+    * [x] I am signing on behalf of myself as an individual and no other person
```

Comment:
Thanks so much for the PR! One small thing: I think you accidentally also included your updates to the contributor agreement *template* (in addition to `miroli.md`). Could you revert that? Aside from hat, everything looks good 👍